### DISCUSSION TITLE
Ability to switch accounts in github destkop (FEEDBACK)

### DISCUSSION THREAD

**Discussion by erenjeagerot** [community/community#34745] [Other Feature Feedback, Questions, & Ideas]
I wish we could switch accounts on github destkop.

![image](https://user-images.githubusercontent.com/43472188/193341670-c3c83035-9980-4726-93fd-7a19e1310475.png)

**Comment by HanaOuerghemmi** [comment]
hello,
check this link maybe can help you 
https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/configuring-and-customizing-github-desktop/setting-a-theme-for-github-desktop
